src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_Update.java

```diff
@@ -63,33 +56,27 @@ public void run() {
                 final byte[] pluginBin;
 
                 try {
-                    pluginBin = Updater.downloadUpdatedJar();
+                    plugin.getUpdateWatcher().getUpdater().install(plugin.getUpdateWatcher().getUpdater().update(plugin.getUpdateWatcher().getUpdater().getCurrentRunning()));
```

Comment:
Same as above, using install().

Reply:
that is design to allow switch different Type of release and install from different source.